viewer/packages/pointclouds/src/PointCloudFactory.ts

```diff
@@ -24,7 +31,11 @@ export class PointCloudFactory {
   async createModel(modelMetadata: PointCloudMetadata): Promise<PotreeNodeWrapper> {
     const { modelBaseUrl, modelIdentifier } = modelMetadata;
 
-    const annotationInfo = await this._annotationProvider.getAnnotations(modelIdentifier);
+    const annotationInfoPromise = this._annotationProvider.getAnnotations(modelIdentifier);
+    const classSchemaPromise = this._classificationsProvider.getClassifications(modelMetadata);
+
+    const annotationInfo = await annotationInfoPromise;
```

Comment:
Clarification: how different it is to assign await in the above statement like
`const annotationInfoPromise = await this._annotationProvider.getAnnotations(modelIdentifier);`

Reply:
The difference is that both `annotationInfoPromise` and `classSchemaPromise` are both network requests that may take some time to finish. So I ensure both of them have been started before waiting for one of them, so that the requests can potentially run in parallel.